plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -667,9 +659,9 @@ protected function get_image( $id, $size = 'thing-thumbnail' ) {
 	 * @since  1.0.0
 	 */
 	public function register_image_sizes() {
-		if ( function_exists( 'add_image_size' ) ) {
-			//add_image_size( $this->post_type . '-thumbnail', 150, 9999 ); // 150 pixels wide (and unlimited height)
-		}
+		// if ( function_exists( 'add_image_size' ) ) {
+			// add_image_size( $this->post_type . '-thumbnail', 150, 9999 ); // 150 pixels wide (and unlimited height)
+		// }
 	} // End register_image_sizes()
```

Comment:
If all of this function is to be commented out should it just be removed? @Kenshino ?